sc2/bot_ai_internal.py

```diff
@@ -227,33 +368,42 @@ def _find_expansion_locations(self) -> None:
         for resources in resource_groups:
             # Possible expansion points
             amount = len(resources)
-            # Calculate center, round and add 0.5 because expansion location will have (x.5, y.5)
-            # coordinates because bases have size 5.
-            center_x = int(sum(resource.position.x for resource in resources) / amount) + 0.5
-            center_y = int(sum(resource.position.y for resource in resources) / amount) + 0.5
-            possible_points = (Point2((offset[0] + center_x, offset[1] + center_y)) for offset in offsets)
-            # Filter out points that are too near
-            possible_points = (
-                point
-                for point in possible_points
-                # Check if point can be built on
-                if self.game_info.placement_grid[point.rounded] == 1
-                # Check if all resources have enough space to point
-                and all(
-                    point.distance_to(resource) >= (7 if resource._proto.unit_type in geyser_ids else 6)
-                    for resource in resources
-                )
-            )
+            # this check is needed for TorchesAIE where the gold mineral wall has a
+            # unit type of `RichMineralField` so we can only filter out by amount of resources
+            if amount > 12:
+                logger.warning(f"Found resource group of size {amount}, possible mineral wall? Skipping this one.")
```

Comment:
```suggestion
                logger.debug(f"Found resource group of size {amount}, possible mineral wall? Skipping this one.")
```
I think this function was only called once at the start of the game? However, each bot playing on this map or a map with a mineral wall will receive such a loguru entry. Is this desired behavior?

Reply:
Yeah, I wasn't too sure how to handle it. Should the logging behavior be removed?

Reply:
Yes, I think complete removal of logging would be even better.